src/main/ts/components/Editor.tsx

```diff
@@ -152,12 +151,23 @@ export class Editor extends React.Component<IAllProps> {
     if (this.editor !== undefined) {
       // typescript chokes trying to understand the type of the lookup function
       configHandlers(this.editor, prevProps, this.props, this.boundHandlers, (key) => this.props[key] as any);
+      if (prevProps.onEditorChange === undefined) {
+        if (this.props.onEditorChange !== undefined) {
+          // added onEditorChange
+          this.editor.on('change keyup setcontent', this.handleEditorChange);
+        }
+      } else {
+        if (this.props.onEditorChange === undefined) {
+          // removed onEditorChange
+          this.editor.off('change keyup setcontent', this.handleEditorChange);
+        }
+      }
```

Comment:
```suggestion
      if (prevProps.onEditorChange === undefined && this.props.onEditorChange !== undefined) {
          // add onEditorChange
          this.editor.on('change keyup setcontent', this.handleEditorChange);
      } else if (this.props.onEditorChange === undefined) {
          // remove onEditorChange
        this.editor.off('change keyup setcontent', this.handleEditorChange);
      }
```

Reply:
I reformatted it and then realized that the behavior of the value prop also depends on handleEditorChange so I changed it to be bound when either or both is in use.